File: app/src/main/java/com/bmsr/scaleheaderdemo/launchmodel/ActivityC.java
package com.bmsr.scaleheaderdemo.launchmodel;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.bmsr.scaleheaderdemo.R;

public class ActivityC extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_a);
        Button btn = findViewById(R.id.btn);
        btn.setText("cccc");
        btn.setOnClickListener(view-> {
            view.getContext().startActivity(new Intent(view.getContext(), ActivityB.class));
        });
        Toast.makeText(this,"测试页面---> cccc",Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/bmsr/scaleheaderdemo/ActivityC.kt
package com.bmsr.scaleheaderdemo

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_c.*
import kotlinx.android.synthetic.main.activity_main.*

class ActivityC :AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_c)
        val list = mutableListOf("测试数据")
        for (i in 1..40) {
            list.add("测试数据---> " + i)
            Log.i("wdd", "list data " + list[i])
        }
        recycler_view.adapter = TestAdapter(list)
        recycler_view.layoutManager = LinearLayoutManager(this);

    }
}